HW4/src/reverseNumber.js

```diff
@@ -0,0 +1,5 @@
+function reverseNumber(x) {
+    var result = x.toString().split("").reverse().join("")
+    return Number(result);
+}
```

Comment:
not a valid result with a negative value

Reply:
+